glass/src/lib/native/cpp/Window.cpp

```diff
@@ -57,12 +57,22 @@ void Window::Display() {
     ImGui::PushStyleVar(ImGuiStyleVar_WindowPadding, m_padding);
   }
 
-  std::string label;
+  std::string name = m_name;
```

Comment:
```suggestion
  std::string* name = &m_name;
```

And update the remainder to match.  There's no reason to copy the string here.